Bubble_Sort/bubble_sort.dart

```diff
@@ -0,0 +1,96 @@
+import 'dart:io';
+
+class BubbleSort 
+{
+    List<int> unsorted;
+    BubbleSort(this.unsorted);
+
+    //add array that you want to sort
+    void addArray(int count, List<int> list) 
+    {
+		list = [];
+		unsorted = list;
```

Comment:
Something happened to line 11, 12.
Tabs ? Please fix this.

Reply:
Done. Please check